Description: removes "Fork Me At GitHub" ribbons from documentation
	As far as I can tell, these ribbons do not fall under a specific licence. The base files have been 
	released by GitHub under the MIT licence, but no licence is referred to for the originals. In any case, 
	this ribbon is not originating from the GitHub site nor from the author's own site.
Author: Caitlin Matos <caitlin.matos@zoho.com>
Last-Update: 2014-07-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/_themes/attic/sidebarusefullinks.html
+++ b/docs/_themes/attic/sidebarusefullinks.html
@@ -1,10 +1,7 @@
-<a href="https://github.com/jborg/attic"><img style="position: fixed; top: 0; right: 0; border: 0;"
-  src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" alt="Fork me on GitHub"></a>
-
 <h3>Useful Links</h3>
 <ul>
   <li><a href="https://pypi.python.org/pypi/Attic">Attic @ PyPI</a></li>
   <li><a href="https://github.com/jborg/attic">Attic @ github</a></li>
   <li><a href="https://attic-backup.org/">Online Documentation</a></li>
   <li><a href="https://github.com/jborg/attic/issues">Issue Tracker</a></li>
-</ul>
\ No newline at end of file
+</ul>
